src/Microsoft.Data.SqlClient/netcore/src/Resources/SR.resx

```diff
@@ -292,7 +292,7 @@
     <value>{0} requires the command to have a transaction when the connection assigned to the command is in a pending local transaction.  The Transaction property of the command has not been initialized.</value>
   </data>
   <data name="ADP_OpenReaderExists" xml:space="preserve">
-    <value>There is already an open DataReader associated with this {0} which must be closed first.</value>
+    <value>There is already an open DataReader associated with this Command which must be closed first.</value>
```

Comment:
Not sure why pulling from master didn't sync this one.. anyway this is a fix from before in the master branch, this change needs to be reverted back to `{0}`. I think the same applies to line 634.

Reply:
I don't know why this is not picked up when I merge `master` into `this_branch`. I will bring in the change manually.